### DISCUSSION TITLE
[RFC] A CLI option to ignore private workspaces

### DISCUSSION THREAD

**Discussion by jameslnewell** [vercel/turborepo#7258] [Ideas]
### Describe the feature you'd like to request

I'd like a CLI option to ignore "private" workspaces

`"private": true`

### Describe the solution you'd like

When some option is passed to the CLI, workspaces with `"private": true` are ignored.

I don't really have a preference around how the filter is passed to the CLI:
- [`lerna` had `--no-private`](https://www.npmjs.com/package/@lerna/filter-options)
- to make it flexible and support future requests from people you could do something similar to `jq` syntax e.g. `--filter=<.private == true>`

🤷 

### Describe alternatives you've considered

At the moment I'm passing a filter argument for each private workspace which gets unwieldy very quickly and could become out-of-sync with the value in `package.json` files at any time 

e.g. `--filter="!privateWorkspace1" --filter="!privateWorkspace2" ...`

**Comment by sppatel** [comment]
+1 I am in need of this as well.

**Comment by mehulkar** [comment]
Good idea! We can consider this
